object.go

```diff
@@ -34,9 +34,27 @@ func (c Color) MarshalText() ([]byte, error) {
 	return []byte(c), nil
 }
 
+type DatabaseMention struct {
+	ID ObjectID `json:"id"`
+}
+
+type PageMention struct {
+	ID ObjectID `json:"id"`
+}
+
+type Mention struct {
+	Type            ObjectType       `json:"type,omitempty"`
+	DatabaseMention *DatabaseMention `json:"database,omitempty"`
+	PageMention     *PageMention     `json:"page,omitempty"`
+	User            *User            `json:"user,omitempty"`
+	Date            *DateObject      `json:"date,omitempty"`
+}
+
 type RichText struct {
 	Type        ObjectType   `json:"type,omitempty"`
-	Text        Text         `json:"text"`
+	Text        *Text        `json:"text,omitempty"`
+	Mention     *Mention     `json:"mention,omitempty"`
+	Equation    *Equation    `json:"equation,omitempty"`
```

Comment:
And what about [Link preview](https://developers.notion.com/reference/rich-text#link-preview-mentions) mentions?
May be we can add it here also?

Reply:
I tried but I was not able to create link preview mentions from Notion. Also, the documentation doesn't provide much information about Link preview schema and how the link preview JSON structure look like.